test/value-set.js

```diff
@@ -0,0 +1,162 @@
+/**
+* Copyright (c) 2017-present, Jembi Health Systems NPC.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict'
+
+const env = require('./test-env/init')()
+const server = require('../lib/server')
+const tap = require('tap')
+const request = require('request')
+
+const basicValueSetTest = (t, test) => {
+  env.initDB((err, db) => {
+    t.error(err)
+
+    server.start((err) => {
+      t.error(err)
+
+      test(db, () => {
+        env.clearDB((err) => {
+          t.error(err)
+          server.stop(() => {
+            t.end()
+          })
+        })
+      })
+    })
+  })
+}
+
+tap.test('ValueSet Resource', { autoend: true }, (t) => {
+  t.test('should return ValueSets matching the url parameter', (t) => {
+    const valueSet = {
+      resourceType: 'ValueSet',
+      active: true,
+      url: 'http://test.valueset.org/valueset1',
+      codeSystem: {
+        system: 'hearth:sssa:procedure-codes',
```

Comment:
sorry i missed this one earlier, lets update the "sssa" reference and change it to "valueset" or something non project specific